DECISION-MAKING.md

```diff
@@ -0,0 +1,151 @@
+`Last update: 2018-07`
+
+# Service owners
+
+High level service and product owners for Mozilla IAM.
+
+* Capability owner: @jeffbryner
+  * Product owner: @gdestuynder
+    * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+    * Service Integration (CIS): @akrug
+    * LDAP: @jdow
+    * DuoSecurity: @gdestuynder
+    * Mozillians.org: @hmitsch
+    * WorkDay: HRIS
+
+## Architects
+
+* @gdestuynder (IAM, Security)
+* @mvk-mozilla (IT)
+
+## Program Managers
+
+* @hmitsch
+
+# Modules owners
+
+This document list which team or persons are responsible for which component (module) of the Mozilla IAM stack. This is
+a more detailed model.
+
+This is mainly useful for decision making and is loosely following the [Firefox Module
+system](https://wiki.mozilla.org/Modules) as well as
+[RACI](https://en.wikipedia.org/wiki/Responsibility_assignment_matrix).
+
```

Comment:
Maybe a role definition section? (instead of defining the role in each component ( Owner (accountable,decision maker) )